packages/calling-stateful-client/src/StatefulCallClient.test.ts

```diff
@@ -166,7 +165,6 @@ describe('Stateful call client', () => {
         )
       ).toBe(true);
       expect(Object.keys(client.getState().calls[callId]?.remoteParticipantsEnded ?? {}).length).toBe(1);
-      // FIXME: There should be only one event triggered here.
```

Comment:
There are two events here because two changes are made: (1) `remoteParticipants` and (2) `remoteParticipantsEnded`.

We could have batched those two changes in a single event, but unlike the other places in this commit, this isn't a bug in how `stateChanged` event is fired.